docs/pages/components/ActionMenu.mdx

```diff
@@ -0,0 +1,424 @@
+        <ActionListItem selected={selected === 'Super Large'}>Super Large</ActionListItem>
+      </ActionList>
+    </ActionMenu>
+  );
+}
+```
+
+### Multiple Selection
+
+`ActionMenu` can also support the selection of multiple action items, with the `selectionType="multiple"` prop:
+
+```jsx example
+function MultiSelectionExample() {
+  const [selected, setSelected] = useState(['Lettuce', 'Tomato']);
+  
+  const handleAction = (key) => {
+    if (selected.includes(key)) {
+      setSelected(selected.filter(value => value !== key))
+    } else {
+      setSelected([...selected, key])
+    }
+  };
+  
+  return (
+    <ActionMenu
+      trigger={
+        <Button variant="secondary">
+          {selected.length && selected.length <= 3 
+            ? `Sandwich Ingredients: ${selected.join(', ')}` 
+            : `Sandwich Ingredients: ${selected.length} selected...`
+          }
+          <Icon type="chevron-down" />
+        </Button>
+      }
+    >
+      <ActionList selectionType="multiple" onAction={handleAction}>
+        <ActionListGroup label="Protein">
+          <ActionListItem selected={selected.includes('Bacon')}>Bacon</ActionListItem>
+          <ActionListItem selected={selected.includes('Ham')}>Ham</ActionListItem>
+          <ActionListItem selected={selected.includes('Turkey')}>Turkey</ActionListItem>
+          <ActionListItem selected={selected.includes('Tofurky')}>Tofurky</ActionListItem>
+        </ActionListGroup>
+        <ActionListGroup label="Cheese">
+          <ActionListItem selected={selected.includes('Gruyere')}>Gruyere</ActionListItem>
+          <ActionListItem selected={selected.includes('Gouda')}>Gouda</ActionListItem>
+          <ActionListItem selected={selected.includes('Pepperjack')}>Pepperjack</ActionListItem>
+          <ActionListItem selected={selected.includes('Swiss')}>Swiss</ActionListItem>
+        </ActionListGroup>
+        <ActionListItem selected={selected.includes('Lettuce')}>Lettuce</ActionListItem>
+        <ActionListItem selected={selected.includes('Tomato')}>Tomato</ActionListItem>
+        <ActionListItem selected={selected.includes('Onions')}>Onions</ActionListItem>
+      </ActionList>
+    </ActionMenu>
+  );
+}
+```
+
+### Mixed Selections
+
+An `ActionMenu` can also contain groupings that have different sets of selections.
+
+```jsx example
+function MixedSelectionExample() {
+  const [difficulty, setDifficulty] = useState('Standard');
+  const [colors, setColors] = useState(['Orange']);
+
+  const handleDifficultyAction = (key) => {
+    setDifficulty(key);
+  }
+
+  const handleColorsAction = (key) => {
+    if (colors.includes(key)) {
+      setColors(colors.filter(value => value !== key))
+    } else {
+      setColors([...colors, key])
+    }
+  }
+
+  return (
+    <ActionMenu
+      trigger={
+        <Button variant="secondary">
+          Setup
+          <Icon type="chevron-down" />
+        </Button>
+      }
+    >
+      <ActionList selectionType="multiple">
```

Comment:
Under the "Experiments" header in Figma, it says 
<img width="273" alt="Screenshot 2025-05-08 at 9 25 35 PM" src="https://github.com/user-attachments/assets/4624430b-2032-49ba-b956-2cbf3d78504e" />


When I was playing around with this example, I realized that the top list was a single select and the bottom list was a multiple select. Is this something that we would add later?

Reply:
I'm not exactly sure what you're asking. The experiment was supposed to be an "add later" thing I think for group descriptions since we opted not to handle that right now.